subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/src/zephyr_fs_mgmt.c

```diff
@@ -28,9 +42,7 @@ fs_mgmt_impl_filelen(const char *path, size_t *out_len)
 	return 0;
 }
 
-int
-fs_mgmt_impl_read(const char *path, size_t offset, size_t len,
-				  void *out_data, size_t *out_len)
+int fs_mgmt_impl_read(const char *path, size_t offset, size_t len, void *out_data, size_t *out_len)
```

Comment:
Style changes like this, the `zephyr_fs_mgmt_truncate` and `zephyr_fs_mgmt_truncate` should at least get separate commit, or PR. Otherwise this just bloats PR diff with changes that are not related to the subject of the change.

Reply:
This was caused by using clang format to try and get rid of the 'do...while' error, should I just revert the changes and ignore that error?